src/app_service/service/community/service.nim

```diff
@@ -507,6 +509,12 @@ QtObject:
     self.events.emit(SIGNAL_COMMUNITY_MEMBERS_CHANGED,
       CommunityMembersArgs(communityId: community.id, members: community.members))
 
+  proc communityTokensChanged(self: Service, community: CommunityDto, prev_community: CommunityDto): bool =
+    let communityTokens = community.communityTokensMetadata
+    let prevCommunityTokens = prev_community.communityTokensMetadata
+    # checking length is sufficient
+    return communityTokens.len != prevCommunityTokens.len
```

Comment:
Will this cover changes on an existing token like for example if it's triggered a remote desctruct or some any other action that can change the internal data of a specific token (no additions or removals in the `communityTokens` list len?

Reply:
We do not handle propagation of burning or minting. There is a bug for this.
Also in community description there are no Total or Remaining numbers.
The update of these values will be propagated differently in the Bug task.